internal/clients/errors.go

```diff
@@ -1,10 +1,16 @@
 package clients
 
-import "errors"
+import (
+	"errors"
+	"fmt"
+)
 
 var (
+	ClientErr = errors.New("HTTP client")
+
 	// Common errors
-	NotFoundErr       = errors.New("backend service returned not found (404) or no data")
+	NotFoundErr = fmt.Errorf("%w: backend service returned not found (404) or no data", ClientErr)
+
 	UnauthorizedErr   = errors.New("backend service returned unauthorized (401)")
 	ForbiddenErr      = errors.New("backend service returned forbidden (403)")
 	Non2xxResponseErr = errors.New("backend service did not return 2xx")
```

Comment:
This was not intended, removing.

Reply:
I think this is still in the commit, right?